tests/Younited.HealthCheck.EntityFrameworkMigrations.Tests/Younited.HealthCheck.EntityFrameworkMigrations.Tests.csproj

```diff
@@ -1,19 +1,20 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net7.0</TargetFrameworks>
+    <TargetFrameworks>net8.0</TargetFrameworks>
```

Comment:
shouldn't we run tests againsst net6.0, net7.0, and net8.0 to ensure the frameworks for which we publish the library are actually tested ?

Reply:
I agree.
My first try "locally" was to add net8. Then I realized net6 was not tested so I decided to fit with existing.
But i will change that.

Reply:
@MartyMcFlaye, we took the opportunity with @GuillaumeGuerra to enhance this part as part of your change. Thanks!